js/script.js

```diff
@@ -26,9 +26,9 @@ async function fetchRepos() {
     loading.classList.remove('hide');
     try {
         let response = await fetch(`https://api.github.com/users/${userName}/repos`)
-        if (!response.ok) {
-            throw new Error('Sorry, the specified user does not exist');
-        }  
+        // if (!response.ok) {
```

Comment:
do not leave commented code in file
If you want to stay some commented code it is better to use this construction:

// TODO: you commets 
but do not stay commented code!